src/libs/ReportUtils.ts

```diff
@@ -7483,6 +7483,23 @@ function isExported(reportActions: OnyxEntry<ReportActions>) {
     return Object.values(reportActions).some((action) => ReportActionsUtils.isExportIntegrationAction(action));
 }
 
+function getApprovalChain(policy: OnyxEntry<Policy>, employeeAccountID: number, reportTotal: number): string[] {
+    const approvalChain: string[] = [];
+
+    // If the policy is not on advanced approval mode, we should not use the approval chain even if it exists.
+    if (!PolicyUtils.isControlOnAdvancedApprovalMode(policy)) {
+        return approvalChain;
```

Comment:
NAB - Eventually we should change this to return the policy's default approver in this case. But not needed at this exact moment probably since we only call this function for advanced approval policies i believe

Reply:
yeah here some cleanup would be fine. i didn't want the function to be used by non advanced policy otherwise we'll have to test every flow